### ISSUE TITLE
Multiscreen configuration

### ISSUE THREAD

**Issue by gpapin** [sindresorhus/wallpaper#5]
Hi, 

I'm wondering how do you handle multiscreen configuration ? 
Are you setting the same image on different screen or can you assign an image as a wallpaper for a particular screen (or desktop in case of osx).

I can identify 3 use-cases :
- Set ```genericWallpaper.jpg``` to screen all screen A,B,C
- Set ```panoramicWallpaper.jpg``` to be a single image covering screen A,B,C
- Set ```wallA.jpg``` and ```wallB.jpg``` to respectively cover screen A and B

**Comment by sindresorhus** [comment]
Currently I'm just setting the first screen. I'm not on a multiscreen config, so wasn't a priority.

Related: https://github.com/sindresorhus/osx-wallpaper/issues/2

**Comment by dorian-marchal** [comment]
I'm on Ubuntu, with a multiscreen configuration and `wallpaper.set` set my wallpaper on both screens (first use-case).

**Comment by sindresorhus** [comment]
@dorian-marchal Yeah, it's hard to preserve the same behaviour cross-platform. It only sets the wallpaper on the first screen on OS X and Windows. No idea if there's any way to do the same on Ubuntu. https://github.com/sindresorhus/linux-wallpaper/blob/master/index.js

**Comment by dorian-marchal** [comment]
I like it this way, I think this the desirable behaviour (unless wallpaper offers a way to change both screens independently).

**Comment by sindresorhus** [comment]
@dorian-marchal Yeah, I guess I can make that default for all platforms for now.

**Comment by kevva** [comment]
@sindresorhus, I don't think you can have different backgrounds on Linux (Ubuntu) on different screens by default, at least not without using an external tool.

**Comment by sindresorhus** [comment]
Support for multiple screens was added for macOS in v4.

**Comment by lqzhgood** [comment]
this App can set multiple screens on Windows
But I won't C language.
Maybe it will help you

https://github.com/Phantom12/Unsplash-Wallpaper-Windows

**Comment by nikolay-borzov** [comment]
Apparently, in order to set wallpapers for multiple monitors images must be combined into one and result wallpaper should be tiled.
See [stack overflow answer](https://stackoverflow.com/a/4468717/1606662)
